cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -0,0 +1,193 @@
+* To select an AWS account that does not have the contract enabled, refer to the first few steps in this tutorial to enable the contract and allow the service charging, which is required for a successful cluster deployment.
+
+. In the following steps, you will specify technical details of the cluster that is to be deployed:
++
+image::rosa-cli-details-18.png[]
+
+* These steps are beyond the scope of this tutorial. See xref:../rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc#rosa-hcp-sts-creating-a-cluster-quickly[Creating ROSA with HCP clusters using the default options] for more details about how to complete the {hcp-title} cluster deployment using the CLI.
+
+== {hcp-title} cluster deployment using the web interface
+
+. A cluster can be created using the web interface by selecting the second option in the bottom section of the introductory *Set up ROSA* page:
++
+image::rosa-deploy-ui-19.png[]
+
+. The first step when creating a ROSA cluster using the web interface is the control plane selection. Make sure the *Hosted* option is selected before clicking the *Next* button:
++
+image::rosa-deploy-ui-hcp-20.png[]
+
+. The next step *Accounts and roles* allows you specifying the infrastructure AWS account, into which the the ROSA cluster will be deployed and where the resources will be consumed and managed:
++
+image::rosa-ui-account-21.png[]
++
+* Click the *How to associate a new AWS account*, if you don not see the account into which you want to deploy the ROSA cluster for detailed information on how to create or link account roles for this association.
+* The `rosa` CLI is used for this.
+* If you are using multiple AWS accounts and have their profiles configured for the AWS CLI, you can use the `--profile` selector to specify the AWS profile when working with the `rosa` CLI commands.
+
+. The billing AWS account is selected in the immediately following section:
++
+image::rosa-ui-billing-22.png[]
++
+* Only AWS accounts that were linked to the Red Hat organization that is currently used will be shown.
+* The specified AWS account will be charged for using the ROSA service, regardless of whether the infrastructure AWS account is linked to it in the same AWS organization.
+* You can see an indicator whether the ROSA contract is enabled for a given AWS billing account or not.
+* In case you would like to use an AWS account that does not have a contract enabled yet, you can either use the _Connect ROSA to a new AWS billing account_ to reach the ROSA AWS console page, where you can activate it after logging in using the respective AWS account by following steps described earlier in this tutorial, or ask the administrator of the AWS account to do that for you.
+
+The following steps past the billing AWS account selection are beyond the scope of this tutorial. 
+//Please have a look at this learning path for more details on how to complete ROSA cluster deployment using the web interface.
```

Comment:
The remark of the limited scope of the tutorial without a tip where to look next (a link) feels a bit vague, but I'm not sure if there's a suitable content to recommend (does not seem to be within the published docs).

Reply:
Updated!